### PR TITLE
Chore: Enforcing a new line after multi-line block statements

### PR DESCRIPTION
**Pull request by jeremypress**

### CODE REVIEW

**Comment by boxcla** [comment]
Verified that @jeremypress has signed the CLA. Thanks for the pull request!

**Comment by priyajeet** [comment]
🔨

File: src/api/uploads/__tests__/PlainUpload-test.js [hunk 1]
```diff
@@ -228,8 +228,11 @@ describe('api/uploads/PlainUpload', () => {
             const folderId = '123';
             const file = {};
             const successCallback = () => {};
+
```
Comment (priyajeet) [inline src/api/uploads/__tests__/PlainUpload-test.js]: This is annoying

Comment (jeremypress) [reply src/api/uploads/__tests__/PlainUpload-test.js]: agreed we should skip one liners. Working on it

**Comment by boxcla** [comment]
# Webpack Bundle Size Github Bot

| Bundle        | Diff | Percentage Diff | Before | After |
| ------------- |:----:|:---------------:|:------:|:-----:|
| Total | 0 B | 0.00% | 0 B | 0 B |

Compared against previously merged PR #434

**Powered by [webpack-bundle-size-github-bot](https://github.com/wenboyu2/webpack-bundle-size-github-bot/)**

**Pull request merged**